src/Sulu/Bundle/SecurityBundle/Controller/ResettingController.php

```diff
@@ -129,6 +118,62 @@ public function resetAction(Request $request)
     }
 
     /**
+     * Returns the sender's email address.
+     *
+     * @param Request $request
+     *
+     * @return string
+     */
+    protected function getSenderAddress(Request $request)
+    {
+        return 'no-reply@' . $request->getHost();
+    }
+
+    /**
+     * @return Translator
+     */
+    protected function getTranslator()
```

Comment:
I'd move that to the very end of the file, since there is already another function like this.